run.sh

```diff
@@ -0,0 +1,21 @@
+# Set & move to home directory
+source set_env.sh
+cd "$SNORKELHOME"
+
+# Make sure the submodules are installed
+git submodule update --init --recursive
```

Comment:
What submodule is this for? Want to make sure this won't update the submodule beyond the version specified by the repo.